shell/components/CommunityLinks.vue

```diff
@@ -66,11 +82,11 @@ export default {
 
 <template>
   <div>
-    <SimpleBox :title="t(title)" :pref="pref" :pref-key="prefKey">
+    <SimpleBox :title="t('customLinks.displayTitle')" :pref="pref" :pref-key="prefKey">
       <div v-for="(value, name) in options" :key="name" class="support-link">
-        <a v-t="name" :href="value" target="_blank" rel="noopener noreferrer nofollow" />
+        <a :href="value" target="_blank" rel="noopener noreferrer nofollow"> {{ name }} </a>
       </div>
-
+      <slot />
```

Comment:
![image](https://user-images.githubusercontent.com/18697775/189913654-814aa2d6-f858-4db6-a8f1-4c50e7ad0694.png)

There's a missmatch between the links in the support page (are external and have an icon to show it) and the external one's from options (which don't have the icon)

Reply:
Discussed this with Ken. All links now would open from a new window. The icon will appear next to the title. `SimpleBox` updated to take a slot now.